### ISSUE TITLE
ISO questions for RGD

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-releases#26]
In AmiGO staging, while RGD lost 17k ISOs, it now has a lot more sources of ISOs: 

Current: 
225895	RGD
133	UniProt
117	ComplexPortal
34	ParkinsonsUK-UCL
33	BHF-UCL

Staging: 
207344	RGD
1127	UniProt
200	ComplexPortal
151	HPA
115	BHF-UCL
81	CAFA
75	MGI
73	ARUK-UCL
56	ParkinsonsUK-UCL
21	CACAO
14	AgBase
11	SynGO
5	HGNC-UCL
2	FlyBase
1	DIBU
1	GO_Central
1	IntAct
1	Reactome
1	WB

However I doubt very much that the ISOs were made by the sources cited, since many of these groups do not produce annotations anymore. Also, the original reference is kept in most cases (but that was already the case, AFAIK).

**Comment by pgaudet** [comment]
Some strange examples: 

1) RGD	61865	Lbp		GO:0033036	PMID:15294986|RGD:9068941	ISO	UniProtKB:P18428	P	lipopolysaccharide binding protein		gene	NCBITaxon:10116	20170221	AgBase	CHEBI:46889|CHEBI:134411|UniProtKB:P08571	


- this is not in AmiGO -current although it's marked as have been annotated in 2017
- the annotation was deleted on 19-Sep-2022, so it's odd that it would appear in this next release

2) RGD:1303008	mutS homolog 5		GO:0000795	"{""relationship"": {""relation"": [{""id"": ""BFO:0000050"", ""label"": ""part of""}], ""id"": ""CL:0000017"", ""label"": ""spermatocyte""}}|{""relationship"": {""relation"": [{""id"": ""BFO:0000050"", ""label"": ""part of""}], ""id"": ""EMAPA:17972"", ""label"": ""EMAPA:17972""}}"	RGD	NCBITaxon:10116	ISO	MGI:1329021	PANTHER:PTHR11361	gene		RGD:1624291	20060817


This annotation is not in MGI: https://www.informatics.jax.org/go/marker/MGI:1913739

**Comment by pgaudet** [comment]
Noting that ISOs assigned by RGD seem to be using the RGD ISO reference: RGD:1624291 , which is what is expected.

**Comment by pgaudet** [comment]
There is a discrepancy between data in QuickGO versus AmiGO: 
eg [2650](https://rgd.mcw.edu/rgdweb/search/search.html?term=2650)

- QuickGO has 'assigned by RGD' (https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=P63138&aspect=cellular_component&evidenceCode=ECO:0000266&evidenceCodeUsage=descendants)
- AmiGO has assigned by ComplexPortal (http://amigo.geneontology.org/amigo/gene_product/RGD:2650)

**Comment by pgaudet** [comment]
Emailed Jennifer Smith at RGD.
